src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -111,10 +121,10 @@ public ChargeableFee getActualCostFee() {
   }
 
   public boolean shouldRefundFees(DateTime lostDateTime) {
-    final DateTime now = ClockManager.getClockManager().getDateTime();
+    final Period overdueMinutesInterval = ofMinutesTillNow(lostDateTime);
 
-    return feeRefundInterval == null || feeRefundInterval
-      .between(lostDateTime, now) <= feeRefundInterval.getDuration();
+    return feeRefundInterval == null
+      || overdueMinutesInterval.compareTo(feeRefundInterval) <= 0;
```

Comment:
My interpretation of this is a check whether the `overdue minutes interval` is less than or equal to `fee refund interval`

Is that a sensible interpretation?

Reply:
yes, it is

Reply:
Could this code be more expressive by making that explicit?

```suggestion
      || overdueMinutesInterval.isLessThanOrEqualTo(feeRefundInterval);
```

That method could use the `compareTo` method internally.

Reply:
Added `isLessThanOrEqualTo` and `isMoreThanOrEqualTo`